### DISCUSSION TITLE
Change configuration of ELK

### DISCUSSION THREAD

**Discussion by Cosmin98-sys** [Security-Onion-Solutions/securityonion#1763] [Unsupported Versions]
Hi!
I'm new with SaltStack and after I've researched on their site for information I've not got too far.
I want to enable Detection tab in Kibana, but for that I need to change a lot of things in Kibana and ElasticSearch configration file. Can you help me with information from where to start ? 

Thank you !

**Comment by TOoSmOotH** [comment]
The first place to start would be the official documentation.

https://docs.securityonion.net/en/2.3/elasticsearch.html

**Comment by Cosmin98-sys** [comment]
Hi!
I've searched in the documentation and on their official site, but it still doesn't work ! When you will enable elastic-auth have any plans to start detection facility too ?

Thank you !
